test/graphql/queries/system_query_test.rb

```diff
@@ -548,6 +548,70 @@ class SystemQueryTest < ActiveSupport::TestCase
       assert_equal @profile1.ssg_version,
                    returned_profiles.dig(0, 'ssgVersion')
     end
+
+    should 'properly sort by compliance score' do
```

Comment:
This test isn't working correctly right now even though the query should be correct. Still looking into why `sortBy` isn't working correctly when sorting anything nested under `testResultProfiles`.